resources/index.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Swedbank Pay Developer Portal – Resources
+opengraph:
+    description: Swedbank Pay Developer Resources
+sidebar:
+  navigation:
+  - title: Checkout
```

Comment:
I don't think Checkout's menu should be on the `resource` pages. :)

Reply:
Agree! It's unnecessary to have that menu in the resources section. Removed